meta/conf/machine/h3ulcb.conf

```diff
@@ -0,0 +1,3 @@
+require xt-distro-machine.inc
+
+MACHINEOVERRIDES =. "rcar:rcar-gen3:"
```

Comment:
yes, someday we will need to do so, but at the moment it's fine: once we start developing real products we most probably will update this to fit actual requirements